package.json

```diff
@@ -146,7 +146,7 @@
     "bignumber.js": "^9.1.2",
     "eventsource": "^2.0.2",
     "randombytes": "^2.1.0",
-    "stellar-base": "10.0.0-beta.4",
+    "stellar-base": "git+https://github.com/stellar/js-stellar-base#master",
```

Comment:
will this pr wait on stellar-base to release first, and pin this to `10.0.0` here?